src/components/FileInputField/__tests__/FileInputField.test.jsx

```diff
@@ -26,7 +28,10 @@ describe('rendering', () => {
   formLayoutProviderTest(<FileInputField {...mandatoryProps} />);
 
   it.each([
-    ...disabledPropTest,
+    // TODO by adam@adamkundrna.cz on 2025-03-03
+    // With the new drop-zone element, we cannot get the input element by label anymore (`getByLabel()`).
+    // We need to find a new way to test this, possibly by removing the wrapping `<label>` element in all components.
+    // ...disabledPropTest,
```

Comment:
Generally speaking it is OK not to use the premade test object (`disabledPropTest`) but to write ones own if the component needs it.

I don't really understand why it does not work though. It seems that label <-> field association works fine in the browser.

Reply:
See my comment about transformation to Playwright above.

I guess that problem might be that [getByLabelText](https://testing-library.com/docs/queries/bylabeltext/) does not detect item correctly when in combination with `within`. Other tests does not use `within`, so that is the difference. What more, `input` is not visible and those queries does not detect hidden elements. For example, getByRole does have special option `hidden` to override the behavior.

But as I say, I would solve it in playwright instead.

Reply:
> But as I say, I would solve it in playwright instead.

:+1: